web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -408,83 +405,6 @@ public static boolean isModulePackageNameInTaskClass(String modulePackageName, S
 	}
 	
 	/**
-	 * Method visibility is package-private for testing
-	 *
-	 * @param mod
-	 * @param realPath
-	 * @should prefix messages with module id
-	 * @should not prefix messages with module id if override setting is specified
-	 */
-	static void copyModuleMessagesIntoWebapp(Module mod, String realPath) {
-		for (Entry<String, Properties> localeEntry : mod.getMessages().entrySet()) {
-			if (log.isDebugEnabled()) {
-				log.debug("Copying message property file: " + localeEntry.getKey());
-			}
-			
-			Properties props = localeEntry.getValue();
-			
-			if (!"true".equalsIgnoreCase(props
-			        .getProperty(ModuleConstants.MESSAGE_PROPERTY_ALLOW_KEYS_OUTSIDE_OF_MODULE_NAMESPACE))) {
```

Comment:
@rkorytkowski Actually modules properties messages are not being shown correctly when module_messages*.properties files are not present in /api/target/classes.
What is more, on the 1.11.x branch they are being deleted from that folder when placed there before build which doesn't happen in the current branch.